libnavigation-ui/src/main/java/com/mapbox/navigation/ui/routealert/RouteAlertModelToll.kt

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.navigation.ui.routealert
+
+import com.mapbox.geojson.Point
+
+/**
+ * Data class for [RouteAlertToll].
+ *
+ * @property coordinate of the toll route alert
+ * @property tollDescription give a description to the toll route alert, it shows under the toll icon
+ */
+data class RouteAlertModelToll(
```

Comment:
In this case, at least the constructor should be internal.

Reply:
Is there any concern of making it public? This model will be used by external developer. We should be good here.

Reply:
If the constructor is public, we won't be able to add new fields to this class without breaking semver. Since the data is generated only on the SDK side, the constructor can be internal as well.

Reply:
It cannot be a `data class` either. Just a:
```suggestion
class RouteAlertModelToll internal constructor(
```

and add all necessary toString/hashCode/equals overrides.

Reply:
> The `RouteAlertToll` has two `onNewRouteTollAlerts` and one of it accepts the `RouteAlertModeToll` as parameter. It enables developers to customize the description.

as I mentioned above, this model will be used by external developers so to address your concern regarding the semver breaking, I use a `builder` pattern for it. Hopefully it makes sense.